docs/API.md

```diff
@@ -394,6 +412,7 @@ tracer.use('express', {
   }
 })
 ```
+[Video guide](https://youtu.be/wtt5Zz7aZhk)
```

Comment:
Generally, I think it's good to have a link contain the title of the destination. For example, by saying "Video guide" or "this", developers clicking on the links won't necessarily know where they're going or if it's fully relevant to them. Accessibility comes into play as well, like when a user with a screen reader tabs to a link, the reader would read out the text of "video guide" or "this".

Instead, something like this would be more explicit:

```markdown
Related video guide: [Modifying a Span Using Hooks](https://youtu.be/wtt5Zz7aZhk).
```

Reply:
You can also use the youtube thumbnail service to embedded the video thumbnail.

### Small version

`[![Modifying a Span Using Hooks](https://img.youtube.com/vi/wtt5Zz7aZhk/hqdefault.jpg)](https://youtu.be/wtt5Zz7aZhk)`

[![Modifying a Span Using Hooks](https://img.youtube.com/vi/wtt5Zz7aZhk/hqdefault.jpg)](https://youtu.be/wtt5Zz7aZhk)

### Large version

`[![Modifying a Span Using Hooks](https://img.youtube.com/vi/wtt5Zz7aZhk/maxresdefault.jpg)](https://youtu.be/wtt5Zz7aZhk)`

[![Modifying a Span Using Hooks](https://img.youtube.com/vi/wtt5Zz7aZhk/maxresdefault.jpg)](https://youtu.be/wtt5Zz7aZhk)

Reply:
Wow thank you both! I'll update it tomorrow ^ ^

Reply:
updated ^ ^